backend/server/src/models/question.rs

```diff
@@ -0,0 +1,131 @@
+use crate::models::error::ChaosError;
+use anyhow::{bail, Result};
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use sqlx::{Pool, Postgres, QueryBuilder, Row};
+
+#[derive(Deserialize, Serialize)]
+pub struct Question {
```

Comment:
Not sure if Deserialize is needed?
Since when question is created, the struct sent from FE would be different to this, with no id and time info.